### ISSUE TITLE
Be able to ignore some properties in hint-compat-api

### ISSUE THREAD

**Issue by CKGrafico** [webhintio/hint#1623]
In the next version of `hint-compat-api` could be a good idea to enable the user to ignore some features via configuration.

e.g: `border-radius`, `box-lines`, `requestAnimationFrame`

Related to: https://github.com/webhintio/rfcs/issues/10
CC/ @molant

**Comment by borgitas21** [comment]
Could you provide the name of the property to define the list of features to ignore? I'm expecting an array that includes a list of name features. Does it make sense for you?
```json
config-schema.json
{
       ...
       "ignoredFeatures": {
            "description": "The feature(s) to ignore by the hints",
            "oneOf": [
                {
                    "type": "array"
                },
                {
                    "type": "string"
                }
            ]
        }
}

**Comment by antross** [comment]
I think just `ignore` will do for the property name.

Also, this should be part of the configuration for the `hint-compat-api` package as opposed to `hint` all-up. As such we won't be editing `config-schema.json`, but instead adding an inline `schema` definition in the `meta` file for each sub-hint in `hint-compat-api`. See [`meta.ts` in `hint-stylesheet-limits`](https://github.com/webhintio/hint/blob/5e02927b3262f25d948e6f9f0ea1c940c7c5f3c0/packages/hint-stylesheet-limits/src/meta.ts#L12) for an example.

Additionally we should include an `enable` list to override our built-in list of ignored features.

Usage would look something like this in `.hintrc` (assuming `cursor` was on our built-in ignore list and you wanted to override that):
```json
{
    ...
    "hints": {
        "compat-api/css": ["error", {
            "enable": ["cursor"]
            "ignore": ["border-radius", "box-lines"],
        }],
        "compat-api/html": ["error", {
            "ignore": ["blink"],
        }],
        ...
    },
    ...
}
```

**Comment by antross** [comment]
Fixed by f110ed51015174f996bd5904061b59901b55b66b
